target/linux/ath79/dts/ar7161_buffalo_wzr-hp-ag300h.dts

```diff
@@ -0,0 +1,287 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	compatible = "buffalo,wzr-hp-ag300h", "qca,ar7161";
+	model = "Buffalo WZR-HP-AG300H/WZR-600DHP";
+
+	aliases {
+		led-boot = &diag;
+		led-failsafe = &diag;
+		led-upgrade = &diag;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	extosc: ref {
+		compatible = "fixed-clock";
+		#clock-cells = <0>;
+		clock-output-names = "ref";
+		clock-frequency = <40000000>;
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		diag: diag {
+			label = "buffalo:red:diag";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+	};
+
+	ath9k-leds {
+		compatible = "gpio-leds";
+		band2g_a {
+			label = "buffalo:amber:band2g";
+			gpios = <&ath9k0 1 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		usb {
+			label = "buffalo:green:usb";
+			gpios = <&ath9k0 3 GPIO_ACTIVE_LOW>;
+			default-state = "off";
```

Comment:
This looks wierd. I don't think USB led is managed by ath9k gpio.

Reply:
Now that you mention this, it really looks weird. But that's how it is defined in [the ar71xx mach file](https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=target/linux/ar71xx/files/arch/mips/ath79/mach-wzr-hp-ag300h.c;hb=HEAD#l76) and it *does* work: `echo none > /sys/class/leds/buffalo\:green\:usb/trigger` turns off the LED next to the usb socket.

Reply:
Hm... It looks like they really use ath9k gpio controller for these leds. I am puzzled what was a reason of doing it.